.github/workflows/test.yaml

```diff
@@ -74,21 +74,38 @@ jobs:
           comment_mode: off
   save-cache:
     name: Save Cache
-    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        os:
+          - ubuntu-22.04
+          - windows-2022
+    runs-on: ${{ matrix.os }}
     steps:
       - name: Check out repository.
         uses: actions/checkout@v3.3.0
       - name: Save Docker images.
         uses: ./
         with:
-          key: docker-cache-test-${{ github.run_id }}-${{ github.run_attempt }}
+          key: docker-cache-test-${{ matrix.os }}-${{ github.run_id }}-${{ github.run_attempt }}
       - name: Build an empty test Docker image.
-        run: docker build --tag empty .
+        run: |
+          if [[ "${{ matrix.os }}" =~ "windows" ]]; then
```

Comment:
`"windows"` should also be single-quoted since there's no interpolation desired.